components/customPostButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Text, TouchableOpacity, StyleSheet} from 'react-native';
+
+interface Props
+{
+    title: string;
+    onPress: () => void;
+}
+
+
+const CustomButton: React.FC<Props> = ({
+    title,
+    onPress,
+  }) => {
+    return (
+        <TouchableOpacity
+            style={styles.button}
+            onPress={onPress}
+        >
+            <Text style={styles.buttonText}>{title}</Text>
+
+        </TouchableOpacity>
+    );
+  };
+  
+  export default CustomButton;
+  
+  const styles = StyleSheet.create({
```

Comment:
Either 4 spaces or 2 spaces for indentation is fine, but don't mix them.
By convention, we use 2 spaces for indentation in DoorGo codebase.

Reply:
Got it, do you use a tool to auto format at the end?

Reply:
i don't use tools. Absolutly, you can use a tool to auto format js/ts code.

Reply:
Used prettier formatter to format code